src/main/java/org/quantumbadger/redreader/activities/OptionsMenuUtility.java

```diff
@@ -148,6 +150,7 @@
 						add(activity, pastMenu, Option.PAST_COMMENTS, true);
 					}
 				}
+				if(optionsMenuItemsPrefs.contains(OptionsMenuItemsPref.SEARCH)) add(activity, menu, Option.SEARCH_COMMENTS, false);
```

Comment:
If user disables search from the settings screen, don't show comment search either